ta4j-core/src/main/java/org/ta4j/core/analysis/criteria/NumberOfLosingTradesCriterion.java

```diff
@@ -0,0 +1,38 @@
+package org.ta4j.core.analysis.criteria;
+
+import org.ta4j.core.TimeSeries;
+import org.ta4j.core.Trade;
+import org.ta4j.core.TradingRecord;
+import org.ta4j.core.num.Num;
+
+/**
+ * Number of losing trades criterion.
+ */
+public class NumberOfLosingTradesCriterion extends AbstractAnalysisCriterion {
+
+    @Override
+    public Num calculate(TimeSeries series, TradingRecord tradingRecord) {
+        long numberOfLosingTrades = tradingRecord.getTrades().stream()
+                .filter(trade -> trade.isClosed())
```

Comment:
Lambda can be replaced with method reference: filter(Trade::isClosed)

Reply:
skip

Reply:
Since there is no performance benefit it sounds more like a matter of taste.

Reply:
But it is code smell, so if you can see it, it should be removed. Why you want to add technical debt?

Reply:
How is this a code smell?